static/tutorial/package-async.md

```diff
@@ -42,7 +42,11 @@ you can get away with `concurrently`/`race`/`Concurrently`, you should.
 
 ```haskell
 #!/usr/bin/env stack
--- stack --install-ghc --resolver lts-5.10 runghc --package async
+{-
+stack script
+  --resolver lts-8.21
+  --package async
```

Comment:
We could go that route it will show the msg `No packages provided, using experimental import parser` to the users. Some might find that scary.

If you are fine with that I will go through them again see if stack can parse the imports in all cases.